guides/miner/ubuntu.mdx

```diff
@@ -49,109 +49,79 @@ In order to run the quai-gpu-miner on Ubuntu and mine valid blocks, you'll need
-    </Warning>
+Run the script with the following command:
 
-     The [quai-gpu-miner](https://github.com/dominant-strategies/quai-gpu-miner) repository contains a [automated script](https://github.com/dominant-strategies/quai-gpu-miner/blob/main/miner_deploy.sh) that can be used to compile and configure the miner.
+<Warning>
+	Running this command will compiling and build the miner. This process may take a while to complete, and requires around 10gb of RAM.
+</Warning>
 
-     To download the script, run the following command:
+```bash
+sudo ./deploy_miner.sh
+```
 
-     ```bash
-     wget https://raw.githubusercontent.com/dominant-strategies/quai-gpu-miner/refs/heads/main/miner_deploy.sh
-     ```
+This will create an `output` directory with the built `quai-gpu-miner` binary in it. Make the `quai-gpu-miner` file executable:
 
-     This script will install the following build dependencies and build the latest version of the miner:
+```bash
+sudo chmod +x output/quai-gpu-miner
+```
 
-     - `git`
-     - `cmake`
-     - `build-essential`
-     - `mesa-common-dev`
-     - [Nvidia CUDA Toolkit v12.6](https://developer.nvidia.com/cuda-12-6-0-download-archive)
+### Dependencies and Drivers
 
-     Make the `miner_deploy.sh` file executable:
+Now that we've installed and built the miner, we need to make sure our system and drivers are up to date.
 
-     ```bash
-     chmod +x miner_deploy.sh
-     ```
+```bash
+sudo apt update && sudo apt upgrade -y
+```
 
-     To compile and configure the miner, run the script with the following command:
+Then, install NVIDIA drivers:
 
-     ```bash
-     sudo ./miner_deploy.sh
-     ```
+```bash
+# if you're running on a non-server OS
+sudo apt install nvidia-driver-535
 
-     This will create an `output` directory with the built `quai-gpu-miner` binary in it. Make the `quai-gpu-miner` file executable:
+# if you're running on a server OS
+sudo apt install nvidia-driver-535-server
+```
 
-     ```bash
-     chmod +x output/quai-gpu-miner
-     ```
+To apply the NVIDIA drivers updates, restart your machine with:
 
-  </Tab>
-</Tabs>
+```bash
+sudo reboot
+```
 
 ## Configure And Run
 
-To run the miner, you'll need a quai-stratum-proxy to connect to. Visit the [quai-stratum-proxy](/guides/client/stratum) docs for information on how to install and configure it. The proxy configuration will determine which shard your gpu-miner is running on and the address payouts are awarded to.
+To run the miner, you'll need a Stratum proxy to connect to. Visit the [quai-stratum-proxy](/guides/client/stratum) docs for information on how to install and configure it. The proxy configuration will determine which shard your gpu-miner is running on and the address payouts are awarded to.
 
-First, you'll need to obtain the IP Address and port your proxy is running on. The default port is `3333`.
+First, you'll need to obtain the IP Address and port your proxy is running on from the Stratum proxy logs. The default port is `3333`.
```

Comment:
Same comment here about localhost

Reply:
Fixed in [cea9168](https://github.com/dominant-strategies/quai-docs/pull/59/commits/cea91684e5c1365b50a9450a0696455bdcf0a0ef)